itest_src/datasets/case01/queries/1084_not_like.sql

```diff
@@ -1,7 +1,3 @@
 -- test for NOT LIKE
 
-
--- pragma sortresult
```

Comment:
Are you sure these `--pragma sortresults` comments are not needed?  My understanding was they informed the integration test to sort results before comparing qserv result to ref db results, because order of row return is not guaranteed?

Reply:
I have checked the code of Qserv (C++ code) and the integration tests, and I'm not seeing anything that might refer to this. This also doesn't seem like a valid MySQL syntax.

Reply:
It's definitely not mysql syntax, as it is commented out. "pragma" comment is handled by the integration test script,: https://github.com/lsst/qserv/blob/main/src/admin/python/lsst/qserv/admin/itest.py#L78

Reply:
Good catch! Though, it looks like the test is sorting results in the text files 🐱 ```
        if outfile is None:
            return _do(subprocess.PIPE)
        else:
            os.makedirs(os.path.dirname(outfile), exist_ok=True)
            with open(outfile, "w") as f:
                _do(f)
                if self.sort_result:
                    self._sort(outfile)
```
I don't think this is the right approach as we want to be sure Qserv did the right thing.

@fritzm what do you think on this?

Reply:
Why is sorting not "the right thing"? How do you compare result rows that appear in random order?